GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -0,0 +1,62 @@
+﻿using CommandLine;
+using GVFS.Common;
+using System;
+
+namespace GVFS.CommandLine
+{
+    [Verb(ConfigVerbName, HelpText = "Get and set GVFS options.")]
+    public class ConfigVerb : GVFSVerb.ForNoEnlistment
+    {
+        private const string ConfigVerbName = "config";
+        private LocalGVFSConfig localConfig;
+
+        [Value(
+                0,
+                Required = true,
+                Default = "",
+                MetaName = "GVFS config setting name",
+                HelpText = "Name of GVFS config setting that is to be set or read")]
+        public string Key { get; set; }
+
+        [Value(
+                1,
+                Required = false,
+                Default = "",
+                MetaName = "GVFS config setting value",
+                HelpText = "Value of GVFS config setting to be set")]
+        public string Value { get; set; }
+
+        protected override string VerbName
+        {
+            get { return ConfigVerbName; }
+        }
+
+        public override void Execute()
+        {
+            this.localConfig = new LocalGVFSConfig();
+
+            string error = null;
+            bool isRead = string.IsNullOrEmpty(this.Value);
+
+            if (isRead)
+            {
+                string value = null;
+                if (this.localConfig.TryGetConfig(this.Key, out value, out error, tracer: null))
+                {
+                    Console.WriteLine(value);
+                }
+                else
+                {
+                    this.ReportErrorAndExit(error);
+                }
+            }
+            else
+            {
+                if (!this.localConfig.TrySetConfig(this.Key, this.Value, out error, tracer: null))

```

Comment:
Turns out that this is case sensitive, and I think that'll be surprising for users. Check out the contents of my config file:

```
C:\Repos\VFSForGit\src>type "C:\ProgramData\GVFS\gvfs.config"
A {"Key":"upgrade.ring","Value":"slow"}
A {"Key":"Upgrade.Ring","Value":"fast"}
```

Reply:
good catch. Thank you!